src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/LambdaConversionResult.cs

```diff
@@ -13,6 +13,7 @@ internal enum LambdaConversionResult
         MismatchedReturnType,
         MismatchedParameterType,
         MismatchedParameterDefaultValue,
+        MismatchedParamsArray,
```

Comment:
This entry can be removed

Reply:
Good catch, removed: 82a9cd87b94b936c178a8f75ff953ffea8972268